codegen.yml

```diff
@@ -8,6 +8,16 @@ generates:
     plugins:
       - "typescript"
       - "typescript-operations"
+    # config:
+    #   scalars:
+    #     BigDecimal: "bignumber.js#BigNumber"
```

Comment:
We can properly import and set the `BigNumber` type @kkennis but we will need to deserialize string into BigNumber on the response. If we need to do this, we'll have to be consistent, and also update some types in other places.